### PR TITLE
Mysterious pool

### PR DESCRIPTION
**Pull request by hendercine**
Added a new option and link to run_out_the_door.md, then added a new page, mysterious-pool,md that the link references. Note that there is a redundant line at the top of the run_out_the_door.md that I started to edit but cannot in order to auto-merge.

### CODE REVIEW

**Comment by hendercine** [comment]
please review this @casey-collab.

File: english/run_out_the_door/run_out_the_door.md [hunk 1]
```diff
@@ -2,6 +2,9 @@ You immediately jump up and run out the nearest exit!
 
 You immediately jump up and run out the nearest [exit](../new-exit/new-exit.md)!
 
+You find that a mysterious pool has formed right outside the door and with a splash
+you [fall in](../mysterious-pool/mysterious-pool.md)!
```
Comment (casey-collab) [inline english/run_out_the_door/run_out_the_door.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/run_out_the_door/mysterious-pool/mysterious-pool.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
+You sink to the bottom like a rock and your vision is obscured by millions of
```
Comment (casey-collab) [inline english/run_out_the_door/mysterious-pool/mysterious-pool.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by hendercine** [comment]
@casey-collab I believe I have fixed the issues.

**Comment by casey-collab** [comment]
Looks great! :fire:

**Pull request merged**